ndb/src/google/cloud/ndb/_eventloop.py

```diff
@@ -345,25 +345,70 @@ def get_event_loop():
     raise exceptions.AsyncContextError()
 
 
-def add_idle(*args, **kwargs):
-    raise NotImplementedError
+def add_idle(callback, *args, **kwargs):
+    """Helper function.
 
+    Calls :method:`EventLoop.add_idle` on current event loop.
 
-def queue_call(*args, **kwargs):
-    raise NotImplementedError
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
+            established by :func:`~google.cloud.ndb.async_context`.
+    """
+    loop = get_event_loop()
+    loop.add_idle(callback, *args, **kwargs)
+
+
+def queue_call(delay, callback, *args, **kwargs):
+    """Helper function.
+
+    Calls :method:`EventLoop.queue_call` on current event loop.
+
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
+            established by :func:`~google.cloud.ndb.async_context`.
+    """
+    loop = get_event_loop()
+    loop.queue_call(delay, callback, *args, **kwargs)
 
 
 def queue_rpc(*args, **kwargs):
     raise NotImplementedError
```

Comment:
Overlooked, or just not obvious how to implement it?

Reply:
#6363